src/main/java/de/thomas_oster/liblasercut/LaserProperty.java

```diff
@@ -34,16 +34,16 @@
    * @param name
    * @return 
    */
-  public abstract Object getMinimumValue(String name);
+  Object getMinimumValue(String name);
   
-  public abstract Object getMaximumValue(String name);
+  Object getMaximumValue(String name);
   
-  public abstract Object[] getPossibleValues(String name);
+  Object[] getPossibleValues(String name);
   
-  public abstract LaserProperty clone();
+  LaserProperty clone();
   
   // Please override equals so that it works well
   // Otherwise there is trouble in the GUI when it tries to compare laser settings
   @Override
```

Comment:
Does "@Override" make any sense here?

Reply:
Yes and no. Yes in that it correctly indicates that Object.equals is overriden.
No because overriding Object.equals in an interface makes little sense, especially given that hashCode is not overriden. The contract of Object.equals and Object.hashCode demand that you deal with both.
Overriding equals in an interface makes sense when you can narrow down the type signature, but as it is here it accepts any Object, so it provides no benefit over the original Object.equals.
Overriding equals in an interface does not guarantee that it is overridden in implementing classes.
I am afraid that this is beyond Java's type system.